src/oracle/price_feed.cairo

```diff
@@ -1,46 +1,83 @@
-/// Mock interface used for oracle::get_price_feed_price. 
-use serde::Serde;
+/// Data Types
+/// The value is the `pair_id` of the data
+/// For future option, pair_id and expiration timestamp
+///
+/// * `Spot` - Spot price
+/// * `Future` - Future price
+/// * `Option` - Option price
+#[derive(Drop, Copy, Serde)]
+enum DataType {
+    SpotEntry: felt252,
+    FutureEntry: (felt252, u64),
+    GenericEntry: felt252,
+}
+
+#[derive(Serde, Drop, Copy)]
+struct PragmaPricesResponse {
+    price: u256,
```

Comment:
```suggestion
    price: u128,
```

changed in latest release, shown [here](https://docs.pragmaoracle.com/docs/GettingStarted/Consuming%20Data%20Feed)